metrics/network/latency_kubernetes/latency-network.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
```

Comment:
we could use `[ -n "$DEBUG" ] && set -x` as other scripts do

Reply:
@chavafg yes I can do the change later for that as I will remove the set -x when I enable the CI limits for this measurement but first I want it like this so I can see the general output in the baremetal instance